tests/generics/src/lib.rs

```diff
@@ -1,41 +1,45 @@
 #![allow(dead_code)]
 
-fn dup<T: Clone>(x: T) -> (T, T) {
-    (x.clone(), x.clone())
+fn f() -> bool {
+    vec![1, 2, 3].into_iter().all(|x| x > 3)
 }
 
-fn foo<const LEN: usize>(arr: [usize; LEN]) -> usize {
-    let mut acc = LEN + 9;
-    for i in 0..LEN {
-        acc += arr[i];
-    }
-    acc
-}
+// fn dup<T: Clone>(x: T) -> (T, T) {
```

Comment:
What's with all the commented code here?

Reply:
Hm, that was a quick test I committed without noticing :roll_eyes: 
Thanks for catching that!